src/__tests__/__snapshots__/index.js.snap

```diff
@@ -202,7 +208,7 @@ exports[`with theme prop of padding 10px 1`] = `
   }
 >
   <div
-    className="css-1p841yy"
+    className="css-1p841yy glamorous(div)"
```

Comment:
this is just the `displayName`, so it's just a string that's automatically created for the displayName of components made with `glamorous.div()`. It's just a way to identify components that were made with that API.